learn/platform/welcome.md

```diff
@@ -0,0 +1,106 @@
+---
+title: 概述
+description: 欢迎访问Moonbeam资料库，Moonbeam是Polkadot上兼容以太坊的智能合约平行链。
+---
+
+# 欢迎访问Moonbeam 
+
+![Main Page Banner Image](/images/main-banner.png)
+
+Moonbeam是同时兼容波卡（Polkadot）生态与以太坊（Ethereum）生态的智能合约平行链。本网站为Moonbeam资料库，在这里，无论您是开发人员，信息收集者，或终端用户都可自由浏览关于Moonbeam网络平台的第一手资料。
+
+本网站信息将随着Moonbeam社区的发展而实时更新。 
+
+欢迎您加入Moonbeam社区，期待与您携手打造去中心化跨链世界。 
+
+---
+
+## 什么是Moonbeam？ {: #what-is-moonbeam }
+
+Moonbeam是开发者友好型区块链，可实现完全兼容EVM、Web3的API兼容以及将Moonbeam连接到现有的以太坊网络的网桥。开发者可借助Moonbeam，利用现有的以太坊开发者工具和网络，轻松实现跨链兼容。开发者只需基于自身开发成果稍作修改，即可轻松将现有的Solidity智能合约和DApp前端部署到Moonbeam。
+
+Moonbeam还将成为波卡（Polkadot）网络的平行链。这意味着Moonbeam不仅能从Polkadot中继链中获得共享的安全性，还能同时集成链接到波卡（Polkadot）的其他链（Polkadot需先实现相关功能）。
+
+---
+
+## 如何参与Moonbeam？ {: #how-to-get-started-with-moonbeam }
+
+### 网络 {: #networks }
+
+目前，任选以下一种方式，助您轻松参与Moonbeam构建：
+
+ - 将您的Moonbeam实例构建为[独立节点](/getting-started/local-node/setting-up-a-node/)
+ - [连接](/getting-started/moonbase/connect/)到[Moonbase Alpha TestNet](/networks/moonbase/) 
+
+### 钱包 {: #wallets }
+
+目前，我们成功测试以下钱包可使用Moonbeam网络：
+
+ - [MetaMask](/integrations/wallets/metamask/)
+ - [MathWallet](/integrations/wallets/mathwallet/)
+ - [Ledger](/integrations/wallets/ledger/)
+ - [Trezor](/integrations/wallets/trezor/)
+
+请注意，我们欢迎任何与以太坊定制网络兼容的钱包与Moonbeam兼容！
+
+### 开发工具 {: #tools }
```

Comment:
this section was broken up into two sections... 1) Deploy a Contract and  2) Tools and Integrations

see here: https://github.com/PureStake/moonbeam-docs/commit/73c120a3601c2969cc2308b0bcaa6656427ffbf8#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R47-R67

Reply:
these same comments apply to the README.md

Reply:
fixed